installers/vagrant-dev/radiasoft-download.sh

```diff
@@ -226,12 +190,47 @@ curl $(install_depot_server)/index.sh | \
   bivio_home_env_ignore_git_dir_ownership=$(vagrant_dev_ignore_git_dir_ownership $os) \
   bash -s redhat-dev
 EOF
-    vagrant_dev_post_install
+    vagrant_dev_post_install "$update"
+}
+
+vagrant_dev_modifiers() {
+    if [[ ${vagrant_dev_vm_devbox:=} ]]; then
+        vagrant_dev_no_mounts=1
+        vagrant_dev_no_vbguest=1
+        vagrant_dev_private_net=1
+    elif [[ ${vagrant_dev_barebones:=} ]]; then
+        # allow individual overrides
+        vagrant_dev_no_dev_env=${vagrant_dev_no_dev_env-1}
+        vagrant_dev_no_mounts=${vagrant_dev_no_mounts-1}
+        vagrant_dev_no_vbguest=${vagrant_dev_no_vbguest-1}
+    fi
+    if [[ $_vagrant_dev_host_os == ubuntu ]]; then
```

Comment:
I think we can remove all of these since they are the default

Reply:
Not sure what you mean. Create a new issue explaining.